tools/build_config/build.ps1

```diff
@@ -188,6 +192,43 @@ function Invoke-Package {
   }
 }
 
+function Invoke-Build-Docs {
+  (Get-Content "./build/CPackConfig.cmake" | Where-Object {$_ -match 'CPACK_PACKAGE_FILE_NAME'}) -match '.*"Horace-([^"]+)".*'
```

Comment:
what is this doing?!

Reply:
Pulling the build details from `build/CPackConfig.cmake` i.e. the version/SHA/etc.